scripts/create_istio_system.sh

```diff
@@ -1,4 +1,24 @@
 #!/bin/bash
 istioctl version
-istioctl install --set revision=${RELEASE_NAME} -f /app/config.yaml
-istioctl verify-install --revision ${RELEASE_NAME}
\ No newline at end of file
+if kubectl get --ignore-not-found=true deploy istiod -n istio-system
```

Comment:
Problem with this approach is that you are assuming if the deploy exists then we want to upgrade the control plane version, when really we might just want to update a setting and leave control plane version. Install should be run for only new settings. 

One way would be to check ENV var of TAG and see if that matches the version of current deployment, and if not then you upgrade. 
`kubectl get --ignore-not-found=true deploy istiod -n istio-system  -o=jsonpath='{$.spec.template.spec.containers[*].image}'` then `if [[ $versions == *"$TAG"*  ]]` something like that. 

Alternative would be to turn this into go program and pull in [istioctl code itself](https://github.com/istio/istio/blob/365a94720d5a5290fafeaee85d49ad83f144be1e/pkg/kube/client.go#L368-L410) to check version and do conditionals, then exec out to binary.